tests/components/geonetnz_quakes/test_sensor.py

```diff
@@ -76,12 +79,15 @@
         assert attributes[ATTR_CREATED] == 3
         assert attributes[ATTR_LAST_UPDATE].tzinfo == dt_util.UTC
         assert attributes[ATTR_LAST_UPDATE_SUCCESSFUL].tzinfo == dt_util.UTC
-        assert attributes[ATTR_LAST_UPDATE] == attributes[ATTR_LAST_UPDATE_SUCCESSFUL]
+        assert attributes[ATTR_LAST_UPDATE] == attributes[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)